### PR TITLE
gutenprint: bumped for R1B4

### PR DESCRIPTION
**Pull request by kenmays**
Tested on hrev56588.
- See: https://dev.haiku-os.org/ticket/17836

### CODE REVIEW

File: net-print/gutenprint/gutenprint9-5.3.4.recipe [hunk 1]
```diff
@@ -11,12 +11,13 @@ replacing the Gimp-Print 4.2-based plugin supplied with GIMP 1.2, \
...
-COPYRIGHT="1999-2020 The authors of Gutenprint"
+COPYRIGHT="1999-2022 The authors of Gutenprint"
 LICENSE="GNU GPL v2"
-REVISION="1"
-SOURCE_URI="https://downloads.sourceforge.net/gimp-print/gutenprint-$portVersion.tar.xz"
-CHECKSUM_SHA256="db44a701d2b8e6a8931c83cec06c91226be266d23e5c189d20a39dd175f2023b"
-SOURCE_DIR="gutenprint-$portVersion"
+REVISION="2"
+SrcGitRev="c64d30b9f93dff5d46134d6cb45af177ab1639e4"
+SOURCE_URI="https://github.com/echiu64/gutenprint/archive/$SrcGitRev.tar.gz"
```
Comment (waddlesplash) [inline net-print/gutenprint/gutenprint9-5.3.4.recipe:18]: This is an unofficial mirror. It should not be used as a source in HaikuPorts.

**Comment by waddlesplash** [comment]
We should not be using a mirror at all.

**Comment by kenmays** [comment]
Official repo - core/main developer (see: QMPlay2).

**Comment by waddlesplash** [comment]
It's still a mirror. We should use the sources from the official site.

**Comment by waddlesplash** [comment]
... we should not be using a snapshot build either, and I'm not sure those have stable SHAsums. We should just use the actual release!

**Comment by korli** [comment]
We already use the release 5.3.4 for gutenprint9. What's the point of this PR?

**Comment by kenmays** [comment]
Ok. Closing.

**Comment by kenmays** [comment]
Note: We didn't migrate haiku-core to gutenprint9 yet - as of hrev56588.

**Comment by kenmays** [comment]
@korli - This PR includes commits fixing bug issues with libusb 1.0.26

**Comment by korli** [comment]
> @korli - This PR includes commits fixing bug issues with libusb 1.0.26

Have you references for these? You can eventually download patches from github, for instance, https://github.com/echiu64/gutenprint/commit/c64d30b9f93dff5d46134d6cb45af177ab1639e4.patch

File: net-print/gutenprint/gutenprint9-5.3.4.recipe [hunk 2]
```diff
@@ -42,21 +44,29 @@ REQUIRES_devel="
...
 	cmd:make
+	cmd:pkg_config$secondaryArchSuffix
 	cmd:sed
+	cmd:yacc
 	"
 
 BUILD()
 {
-	runConfigure ./configure --without-cups
+	autogen.sh --without-cups --with-doc=no --disable-static \
```
Comment (korli) [inline net-print/gutenprint/gutenprint9-5.3.4.recipe:64]: autogen.sh isn't needed on release tarballs.

Comment (kenmays) [reply net-print/gutenprint/gutenprint9-5.3.4.recipe]: Resolved.

**Review by korli — CHANGES_REQUESTED** [review]
Please use a release tarball and a source URL for the patch. Please focus on the bug this PR is supposed to fix, otherwise you can directly close this PR.

**Comment by kenmays** [comment]
The source tarball is the same used by Debian and Ubuntu testing.

**Comment by korli** [comment]
Please use a release tarball and a source URL for the patch. 

> The source tarball is the same used by Debian and Ubuntu testing.

It's still a snapshot.

**Comment by kenmays** [comment]
I'll close for now. No critical patches observed.

**Pull request closed without merging**